utilities/transactions/transaction_test.cc

```diff
@@ -4990,7 +4990,7 @@ TEST_P(TransactionTest, DeleteRangeSupportTest) {
           }
           break;
         case WRITE_PREPARED:
-          // Intentional fall-through
```

Comment:
We have a macro for this too: https://github.com/facebook/rocksdb/blob/main/port/lang.h#L10 how about using that macro for now to be consistent with other code and migrate the macro later.